src/main/java/com/commercetools/project/sync/customobject/CustomObjectSyncer.java

```diff
@@ -59,16 +60,20 @@
    * @param customObjectService service that is used for fetching and persisting the last sync
    *     timestamp for delta syncing.
    * @param clock the clock to record the time for calculating the sync duration.
+   * @param syncProjectSyncCustomObjects if true, custom objects generated by the project sync will
```

Comment:
Does it mean if argument is provided in CLI, it would not sync custom objects generated?
Please check this sentence.